### ISSUE TITLE
Make it awesome

### ISSUE THREAD

**Issue by mika76** [amis92/csharp-source-generators#7]
I think the whole term "awesome list" has become the de-facto standard name for lists like these so it might help people finding it if you name it so....

* https://github.com/emijrp/awesome-awesome - list of awesome lists
* https://github.com/topics/awesome-list - github even has a topic

> A list of C# Source Generators (not necessarily awesome), because I haven't found a good list yet.

I think it's awesome and you can always fake it till you make it :wink:

**Comment by amis92** [comment]
I think for now I can just add the `awesome-list` tag for better search, but since I've included "awesome" in repo description, I think it should work for SEO - they won't really understand the "not necessarily" part, will they. :D

**Comment by ironcev** [comment]
I agree the list is definitely awesome :-) @amis92 and I discussed this already here: https://github.com/ironcev/awesome-roslyn/pull/16#issuecomment-707332959

> How do you see the future of the csharp-source-generators list? Should it stay in your opinion as a collection list and serve as a source for the awesome list? If so, we should link it from the awesome list at least. Or you think that having Source Generators on the awesome list serves the purpose? As mentioned, I see the benefits of both approaches.

Since awesome lists are made to be selections, and not collections we thought the best would be to pick the awesome stuff out and add it to the Source Generator section of the Awesome Roslyn list, here the @amis92 opinion: https://github.com/ironcev/awesome-roslyn/pull/16#issuecomment-707392595

I do mu best to closely follow @amis92 great work and take over generators that reached the certain level of maturity as well as articles and tutorials.

**Comment by AraHaan** [comment]
@ironcev you ok with adding my generator to that list as well too? #46

**Comment by ironcev** [comment]
Hi @AraHaan, thanks for contributing the generator! I plan to take a closer look at the generators added recently to the source-generators list. There are quite some generators added which is great to see :-) But it will make it hard to choose a few to take over. In general I am trying to find some that somehow showcase something not seen in already added generators. E.g. you use Roslyn SyntaxFactory to create the code which is the way I would always prefere, while many generators work directly with string concatenation. Is it fin for you if I ping you back once I take a closer look at the other recently added generators?

**Comment by AraHaan** [comment]
Sure

**Comment by TwentyFourMinutes** [comment]
Hey @ironcev, I don't necessarily want to self advertise my library in any way, nor do I think it is yet worth to add it to this or a similar list, as it is currently not in a usable or production ready state. However, I am currently working on generating a full ORM with Source Generators and [I am utilizing the SyntaxFactory API extensivly](https://github.com/TwentyFourMinutes/Venflow/tree/reflow/src/Reflow.Analyzer/Emitters). I just want to say that I fully agree with you. Generating code with string concatenation is not just unreliable, but also error prone, unmaintainable and a pain in the butt to write.
